lix/packages/sdk/src/newLix.ts

```diff
@@ -58,6 +58,14 @@ export async function newLixFile(): Promise<Blob> {
         meta TEXT,
         commit_id TEXT
       ) strict;
+
+      CREATE TABLE conflict (
+        change_id TEXT NOT NULL,
+        conflicts_with_change_id TEXT NOT NULL,
+        reason TEXT,
+        meta TEXT,
+        PRIMARY KEY (change_id, conflicts_with_change_id)
```

Comment:
Unsure about the primary key here and if a conflict shouldn't have an ID. The problem with an ID would be re-generation of a conflict that is already stored in the conflict table.

Reply:
Agree. Changed it https://github.com/opral/monorepo/pull/3067/commits/537d804abbdb1324a9b3f1ad3e4ca1345102679e